### PR TITLE
Mitschriften SWE 04.12.2020

### PR DESCRIPTION
**Pull request by RvNovae**
hehe

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/SWE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/RES/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/PRO/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/NET/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: SWE/README.md [hunk 1]
```diff
@@ -930,3 +937,155 @@ $$f(Qualitätsmaße) \hat{=}Kennzahlen \rightarrow Aufwand=g(LOC) \cdot f_1(Q,Ma
...
+
+$l\rightarrow COCOMO \rightarrow MM,t$ 
+
+$t_{easySW}=2.5(2.4\cdot (\frac{l}{1000})^{1.05})^{0.38}+t_{AA}$
+
+$\rightarrow$ einfache Software in stabiler Umgebung
+
+@Lit: Gleichungen für Unternehmen kalibrieren
+
+> Vorstellbar als Käse $\copyright$
```
Comment (ZeroPointMax) [inline SWE/README.md:1073]: :D

**Review by ZeroPointMax — APPROVED** [review]
👍

File: SWE/README.md (hunk 1 above)
Comment (RvNovae) [reply SWE/README.md:1073]: 😉

**Review by sthinbetween — APPROVED** [review]

**Pull request merged**